app/controllers/merchant_invoices_controller.rb

```diff
@@ -7,5 +7,22 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
     @invoice_items = InvoiceItem.where(invoice_id: @invoice.id)
+    @items = @invoice.items
+  end
+
+  def update
+    @invoice = Invoice.find(params[:id])
+    if params[:status]["packaged"]
+      @invoice_item = InvoiceItem.find(params[:invoice_item_id])
```

Comment:
Line 16, 19, 22 do the same thing. Could we move it out of the if/else statement?